### PR TITLE
Initial Contribution of the sldt-discovery-finder

### PR DESCRIPTION
**Pull request by tunacicek**
* Initial Contribution for sldt-discovery-finder
* Copyright information -> please check the respective notices in the source code.

### CODE REVIEW

File: docs/1-introduction-and-goals.md
Comment (LuLeRoemer) [inline docs/1-introduction-and-goals.md:1]: What about these files?

File: charts/discoveryfinder/templates/_helpers.tpl
Comment (LuLeRoemer) [inline charts/discoveryfinder/templates/_helpers.tpl:1]: Would not commit empty file.

File: charts/discoveryfinder/LICENSE
Comment (LuLeRoemer) [inline charts/discoveryfinder/LICENSE:1]: What's the extra license file for?

**Review by LuLeRoemer — CHANGES_REQUESTED** [review]

File: docs/1-introduction-and-goals.md
Comment (tunacicek) [reply docs/1-introduction-and-goals.md:1]: These files were created for preparation for the documentation task (As structure of the documentation)

File: charts/discoveryfinder/templates/_helpers.tpl
Comment (tunacicek) [reply charts/discoveryfinder/templates/_helpers.tpl:1]: Empty file is removed.

File: charts/discoveryfinder/LICENSE
Comment (tunacicek) [reply charts/discoveryfinder/LICENSE:1]: Extra license is removed.

**Review by LuLeRoemer — APPROVED** [review]
LGTM

**Pull request merged**